lib/option.js

```diff
@@ -117,7 +117,7 @@ class Option {
    */
 
   choices(values) {
-    this.argChoices = values;
+    this.argChoices = values.slice();
     this.parseArg = (arg, previous) => {
       if (!values.includes(arg)) {
```

Comment:
The uses of `values` in the parse function should be replaced with `this.argChoices` now they may be different.